src/ApiService/ApiService/onefuzzlib/Extension.cs

```diff
@@ -329,37 +331,21 @@ private sealed class Settings {
         throw new NotSupportedException($"unsupported OS: {vmOs}");
     }
 
-    public async Async.Task<VMExtensionWrapper> MonitorExtension(AzureLocation region, Os vmOs) {
+    public async Async.Task<VMExtensionWrapper> MonitorExtension(AzureLocation region) {
         var settings = await _context.LogAnalytics.GetMonitorSettings();
         var extensionSettings = JsonSerializer.Serialize(new { WorkspaceId = settings.Id }, _extensionSerializerOptions);
         var protectedExtensionSettings = JsonSerializer.Serialize(new { WorkspaceKey = settings.Key }, _extensionSerializerOptions);
-        if (vmOs == Os.Windows) {
-            return new VMExtensionWrapper {
-                Location = region,
-                Name = "OMSExtension",
-                TypePropertiesType = "MicrosoftMonitoringAgent",
-                Publisher = "Microsoft.EnterpriseCloud.Monitoring",
-                TypeHandlerVersion = "1.0",
-                AutoUpgradeMinorVersion = true,
-                Settings = new BinaryData(extensionSettings),
-                ProtectedSettings = new BinaryData(protectedExtensionSettings),
-                EnableAutomaticUpgrade = false
-            };
-        } else if (vmOs == Os.Linux) {
-            return new VMExtensionWrapper {
```

Comment:
isn't this a required extension?
cc: @nharper285

Reply:
OMS extension is going to be deprecated in 2024, the Azure Monitor Agent for Linux is the recommended path now (which we have installed) https://learn.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-migration